presto-main/src/main/java/com/facebook/presto/operator/ExchangeOperator.java

```diff
@@ -106,6 +109,9 @@ public ExchangeOperator(
         this.exchangeClient = requireNonNull(exchangeClient, "exchangeClient is null");
         this.serde = requireNonNull(serde, "serde is null");
 
+        this.systemMemoryContext = operatorContext.localUserMemoryContext();
```

Comment:
It's better to create a new LocalMemoryContext so we can tag the meomry usage.
operatorContext.newLocalSystemMemoryContext(ExchangeOperator.class.getSimpleName());